pkg/clusterversion/cockroach_versions.go

```diff
@@ -304,6 +304,11 @@ const (
 	// store all KV span configs.
 	SpanConfigurationsTable
 
+	// ZonesTableForSecondaryTenants adds the zones system table for all secondary
+	// tenants. This doesn't affect the system tenant, as the system tenant
+	// already has the zones table.
```

Comment:
Just the first sentence is sufficient, given the version name. Also s/the zones system table/system.zones.